### DISCUSSION TITLE
Is it possible to add support for the "GC9107" driver?

### DISCUSSION THREAD

**Discussion by Nitric-acid-CN** [Bodmer/TFT_eSPI#3015] [New processor/display support or capability request]
Here is the manual for this driver chip. If you could add support for it, I would be very grateful!
[GC9107 DataSheet V1.2.pdf](https://github.com/Bodmer/TFT_eSPI/files/13426834/GC9107.DataSheet.V1.2.pdf)

**Comment by Bodmer** [comment]
See #2072

It appears that the GC9A01 driver works.

**Reply by NeTgHoStKiLlEr** [reply]
I don't see a selector for that driver for the ESP8266, only for ESP32 or LILYGO. Am I missing it? I'm trying to use the 128x128 D1 Mini Shield, with the .85 TFT. It uses the GC9107 or GC9A01 Driver it says on the WEMOS site, but I also tried the ST7789 driver, and had issues with the image being 1 pixel off on the bottom and right side, showing an extra line of un-cleared pixels.

EDIT: I played around with the driver text files for awhile, adding functions in between yours for my missing 128 screen size. Eventually I think I got it working correctly. The offset values that worked for me were colstart = 2, rowstart = 1 for rotations 0 and 1, and flipped values for the other two rotations.
